### PR TITLE
Add implementation info on image service use case

### PR DESCRIPTION
**Pull request by BirgitBader**
@ubep please check and merge @PMJP already agreed on that :-)

### CODE REVIEW

File: apps/use-cases.md [hunk 1]
```diff
@@ -108,7 +117,7 @@ Availability planned: **06/2016**
...
 
-Suggested calls: [`GET` products](page:apps-api-get-shops-shopid-products-information), [`PATCH` products](page:apps-api-patch-shops-shopid-products-productid-information), `POST` products, [`DELETE` products](page:apps-api-delete-shops-shopid-products-productid-information), `GET` customers, `PUT` customers, `POST` customers, [`GET` orders](page:apps-api-get-shops-shopid-orders-information), [`PUT` orders](page:apps-api-put-shops-shopid-orders-orderid-information), `POST` orders
+Suggested calls: [`GET` products](page:apps-api-get-shops-shopid-products-information), [`PATCH` products](page:apps-api-patch-shops-shopid-products-productid-information), `POST` products, [`DELETE` products](page:apps-api-delete-shops-shopid-products-productid-information), [`GET` customers](page:apps-api-get-shops-shopid-customers-customerid-information), `PUT` customers, [`POST` customers](page:apps-api-post-shops-shopid-customers-information), [`GET` orders](page:apps-api-get-shops-shopid-orders-information), [`PUT` orders](page:apps-api-put-shops-shopid-orders-orderid-information), `POST` orders
```
Comment (ubep) [inline apps/use-cases.md]: POST products and PUT orders are not linked

File: apps/use-cases.md [hunk 2]
```diff
@@ -82,15 +88,18 @@ Availability: <i class="fa fa-check"></i>
...
+Suggested calls: [`GET` products](page:apps-api-get-shops-shopid-products-information), [`PATCH` products](page:apps-api-patch-shops-shopid-products-productid-information), [`GET` products/slideshow](page:apps-api-get-shops-shopid-products-productid-slideshow-information), [`POST` products/slideshow](page:apps-api-post-shops-shopid-products-productid-slideshow-information), [`DELETE` products/slideshow{imageName}](page:apps-api-delete-shops-shopid-products-productid-slideshow-imagename-information), [`GET` products/slideshow/sequence](page:apps-api-get-shops-shopid-products-productid-slideshow-sequence-information), [`PUT` products/slideshow/sequence](page:apps-api-put-shops-shopid-products-productid-slideshow-sequence-information)
```
Comment (ubep) [inline apps/use-cases.md]: missing slash in *products/slideshow/{imageName}*

Comment (ubep) [inline apps/use-cases.md]: maybe better: *...endpoint to upload a new image to the slideshow of the product*

File: apps/use-cases.md (hunk 1 above)
Comment (BirgitBader) [inline apps/use-cases.md]: yes ;-)

File: apps/implement-use-case.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+layout: page
+key: apps-implement-use-case
+title: Implementing a use case
+---
+
+## Image Service
+
+To add a new image to a product, this is what you would need to do:
+
+1. Use the [`POST` products/slideshow](page:apps-api-post-shops-shopid-products-productid-slideshow-information) endpoint and post a new image to the slideshow of the product.
```
Comment (ubep) [inline apps/implement-use-case.md]: maybe better: ...endpoint to upload a new image to the slideshow of the product

File: apps/implement-use-case.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+If you want to replace an existing image, delete that image first using the [`DELETE` products/slideshow{imageName}](page:apps-api-delete-shops-shopid-products-productid-slideshow-imagename-information) endpoint.
```
Comment (ubep) [inline apps/implement-use-case.md]: missing slash in products/slideshow/{imageName}

File: apps/use-cases.md (hunk 2 above)
Comment (BirgitBader) [inline apps/use-cases.md]: which line does that refer to?

File: apps/implement-use-case.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+2. After having added a new image to the slideshow, you can assign the image to a product using the [`PATCH` products](page:apps-api-patch-shops-shopid-products-productid-information) endpoint.
+Update the product image by addressing the attribute path `productImage`.
+
+Example:
```
Comment (ubep) [inline apps/implement-use-case.md]: kann man das example so einrücken, dass es punkt 2 der auflistung untergeordnet ist?

Comment (BirgitBader) [inline apps/implement-use-case.md]: Leider nicht, habe ich schon in allen möglichen Varianten probiert. Damit müssen wir so leben.

File: apps/implement-use-case.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+key: apps-implement-use-case
+title: Implementing a use case
+---
+
+## Image Service
+
+To add a new image to a product, this is what you would need to do:
```
Comment (ubep) [inline apps/implement-use-case.md]: add further headline:
### Uploading a new product image

**Pull request merged**